File: staffapp/src/main/java/com/example/staffapp/ui/work/SectionIcons.kt
package com.example.staffapp.ui.work

import androidx.compose.material.icons.Icons
import androidx.compose.material.icons.filled.Analytics
import androidx.compose.material.icons.filled.CalendarMonth
import androidx.compose.material.icons.filled.Dashboard
import androidx.compose.material.icons.filled.EventNote
import androidx.compose.material.icons.filled.Folder
import androidx.compose.material.icons.filled.Group
import androidx.compose.material.icons.filled.Payments
import androidx.compose.material.icons.filled.People
import androidx.compose.material.icons.filled.Settings
import androidx.compose.material.icons.filled.SupportAgent
import androidx.compose.material.icons.filled.Task
import androidx.compose.material.icons.filled.FitnessCenter
import androidx.compose.ui.graphics.vector.ImageVector

object SectionIcons {
    fun forSection(key: String): ImageVector = when (key) {
        "dashboard" -> Icons.Default.Dashboard
        "tasks" -> Icons.Default.Task
        "clients" -> Icons.Default.People
        "schedule" -> Icons.Default.CalendarMonth
        "bookings" -> Icons.Default.EventNote
        "subscriptions" -> Icons.Default.FitnessCenter
        "visits" -> Icons.Default.EventNote
        "analytics" -> Icons.Default.Analytics
        "finance" -> Icons.Default.Payments
        "app_support" -> Icons.Default.SupportAgent
        "trainers" -> Icons.Default.Group
        "documents" -> Icons.Default.Folder
        "settings" -> Icons.Default.Settings
        else -> Icons.Default.Dashboard
    }
}

File: MatskocaAdmin/MatskocaAdmin/Labels/SectionIcons.swift
import SwiftUI

enum SectionIcons {
    static func forSection(_ key: String) -> String {
        switch key {
        case "dashboard": return "square.grid.2x2"
        case "tasks": return "checklist"
        case "clients": return "person.2"
        case "schedule": return "calendar"
        case "bookings": return "calendar.badge.clock"
        case "subscriptions": return "figure.strengthtraining.traditional"
        case "visits": return "calendar.badge.clock"
        case "analytics": return "chart.bar"
        case "finance": return "creditcard"
        case "app_support": return "headphones"
        case "trainers": return "person.3"
        case "documents": return "folder"
        case "settings": return "gearshape"
        default: return "square.grid.2x2"
        }
    }
}
